client/state/exporter/actions.js

```diff
@@ -153,7 +151,7 @@ export function exportStatusFetch( siteId ) {
 			return failure( response );
 		};
 
-		return wpcom.undocumented().getExport( siteId, 0 ).then( success ).catch( failure );
+		return wpcom.req.get( `/sites/${ siteId }/exports/0` ).then( success ).catch( failure );
```

Comment:
Interesting that the export ID is always 0 because only one simultaneous export can ever run.